core/sr-io/without_std.rs

```diff
@@ -59,16 +59,16 @@ pub extern fn oom(_: ::core::alloc::Layout) -> ! {
 /// See also "ext_" functions in sr-sandbox and sr-std
 extern "C" {
 	/// Most of the functions below return fixed-size arrays (e.g. hashes) by writing them into
-	/// memory regions that should be preallocated by module. 
+	/// memory regions that should be preallocated by module.
 	/// Functions that return variable-sized data use host-side allocations. These should be
 	/// manually freed by the module.
-	fn ext_free(addr: *mut u8);
-	
+	pub fn ext_free(addr: *mut u8);
```

Comment:
I think we need to delete this. allocator crate uses its own externs

Reply:
Okay